cloudconfig/templates.go

```diff
@@ -122,25 +90,15 @@ function downloadAndExtractRunner() {
 	# chown {{ .RunnerUsername }}:{{ .RunnerGroup }} -R /home/{{ .RunnerUsername }}/actions-runner/ || fail "failed to change owner"
 }
 
-CACHED_RUNNER=$(getCachedToolsPath)
-if [ -z "$CACHED_RUNNER" ];then
+RUN_HOME="/home/{{ .RunnerUsername }}/actions-runner"
```

Comment:
the path defined in this variable is also used in `downloadAndExtractRunner` (and probably other places. Could you define the `RUN_HOME` variable at the top and just use it throughout the template?